config/locales/en.yml

```diff
@@ -1196,8 +1196,7 @@ in_person_proofing.body.barcode.location_details: Location details
 in_person_proofing.body.barcode.questions: Questions?
 in_person_proofing.body.barcode.retail_hours: Retail hours
 in_person_proofing.body.barcode.retail_hours_closed: Closed
-in_person_proofing.body.barcode.return_to_partner_html: You may now %{link_html} to complete any next steps you can access until your identity has been verified.
```

Comment:
Looks like `in_person_proofing.body.barcode.return_to_partner_html` needs to be updated in idv_step_helper -